src/lib/utils/interfaces/team.interface.ts

```diff
@@ -48,6 +48,7 @@ export interface TeamServiceMethods {
   ) => unknown;
   removeUserFromWorkspace?: (workspaceId: string, userId: string) => unknown;
   disableNewInviteTag?: (userId: string, teamId: string) => Promise<Team>;
+  updateTeam: (teamId: string, team) => Promise<unknown>;
```

Comment:
instead of unknown can we use void | updateTeamDto

Reply:
resolved